coding/python.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: base
+---
+
+# Python code standards
+
+This document outlines the rules for writing Python code across all our Projects.
+
+## Code style and formatting
+
+All Code should be written in accordance with [PEP 8](https://pep8.org/).
```

Comment:
I think maybe it's worth being clear about which points here are absolute requirements and which are strong suggestions - maybe with a nod towards https://tools.ietf.org/html/rfc2119.

So, in this case I think we can say: 

```suggestion
All code must conform to [PEP 8](https://pep8.org/).
```